EpicLoot/Adventure/BountyTarget.cs

```diff
@@ -61,11 +61,13 @@ private void OnDeath()
         public void Initialize(BountyInfo bounty, string monsterID, bool isAdd)
         {
             _zdo.Set(BountyIDKey, bounty.ID);
-
-            var pkg = new ZPackage();
-            bounty.ToPackage(pkg);
-            pkg.SetPos(0);
-            _zdo.Set(BountyDataKey, pkg.GetBase64());
+            if (ZNet.instance.IsServer() && ZNet.instance.IsDedicated() || !ZNet.instance.IsServer() && !ZNet.instance.IsDedicated())
```

Comment:
What is this actually doing? The operator precedence here means this does something that you probably didn't mean to do?